README.md

```diff
@@ -156,10 +156,10 @@ HOF-bootstrap accepts the following options so a developer can customise element
 
 ### Options
 
-- `views`: Location of the base views relative to the root of your project. Defaults to 'views'. Set `views` to `false` if not present.
+- `views`: Location of the base views relative to the root of your project. Defaults to `__dirname + '/views'`.
+- `fields`: Location of the common fields relative to the root of your project. Defaults to `__dirname + '/fields'`.
+- `translations`: Location of the common translations relative to the root of your project. Defaults to `./translations`.
```

Comment:
I noticed you removed `Set `views` to `false` if not present.` So what do you do now if your views are not present?